src/components/Quote.vue

```diff
@@ -283,7 +283,7 @@ export default {
 	}
 
 	&.quote-own-message::before {
-		background-color: var(--color-primary-default);
```

Comment:
Shouldn't it be aligned with your theming as well?

https://github.com/nextcloud/server/blob/85ff1449d33df76db17d741e46485799e7121977/apps/theming/lib/Themes/CommonThemeTrait.php#L58

Reply:
No, default is default blue
The other option would have been `--color-primary` but that could be white on white, etc. So we have to use element here